homeassistant/components/tomorrowio/__init__.py

```diff
@@ -339,7 +340,6 @@ def __init__(
         self._config_entry = config_entry
         self._attr_device_info = DeviceInfo(
             identifiers={(DOMAIN, self._config_entry.data[CONF_API_KEY])},
-            name=INTEGRATION_NAME,
```

Comment:
With this change, the device name is equal to the config entry title. So if someone has 2 locations set up, they can rename the config entry and the device and entities will change.